prime-router/src/main/kotlin/common/JacksonMapperUtilities.kt

```diff
@@ -51,6 +54,27 @@ object JacksonMapperUtilities {
         enable(SerializationFeature.INDENT_OUTPUT) // Pretty print JSON output
     }.run { setTimeZone(TimeZone.getTimeZone(Environment.rsTimeZone)) }
 
+    private val yamlModule = KotlinModule.Builder()
+        .withReflectionCacheSize(512)
+        .build()
+
+    val yamlMapper: ObjectMapper = ObjectMapper(YAMLFactory())
+        .registerModule(yamlModule)
+        .registerModule(JavaTimeModule()).disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+        .setSerializationInclusion(JsonInclude.Include.NON_NULL)
+        .setSerializationInclusion(JsonInclude.Include.NON_EMPTY)
+
+    val customSerializersMapper: ObjectMapper = jsonMapper {
```

Comment:
Can we rename this to something clearer?  How would I know to use this over `allowUnknownsMapper`?

Reply:
put comments around the mapper definition

Reply:
renamed and re-tested